content/node-runners/postinstall/index.md

```diff
@@ -0,0 +1,195 @@
+
+**TequilAPI** is a REST API that allows you to manipulate a Mysterium node multiple ways.
+
+```
+curl http://localhost:4050/healthcheck
+
+{
+    "uptime": "47m55.616006453s",
+    "process": 5490,
+    "version": "0.0.0-dev",
+    "buildInfo": {
+        "commit": "99d37edc952b736e3fad069f56e7d968276634a8",
+        "branch": "master",
+        "buildNumber": "4755"
+    }
+}
+```
+
+**CLI** is a command line interface that allows you manipulate a Mysterium node using just your terminal.
+It can be started using a `myst cli` command:
+
+```
+myst@raspberrypi:~ $ myst cli
+Mysterium Node
+  Version: 0.0.0-dev
+  Build info: Branch: master. Build id: 4755. Commit: 99d37edc952b736e3fad069f56e7d968276634a8
+...
+» healthcheck
+[INFO] Uptime: 50m25.039797872s
+[INFO] Process: 5490
+[INFO] Version: 0.0.0-dev
+[INFO] Branch: master. Build id: 4755. Commit: 99d37edc952b736e3fad069f56e7d968276634a8
+»
+```
+
+#### Identities
+
+To get a list of Mysterium node identities you can use the following commands:
+
+```
+curl http://localhost:4050/identities
+
+{
+    "identities": [
+        {
+            "id": "0x041e42becaa3a6f92e155a7a5cab62c49dcdc578"
+        }
+    ]
+}
+```
+
+```
+myst@raspberrypi:~ $ myst cli
+...
+» identities list
+[+] 0x041e42becaa3a6f92e155a7a5cab62c49dcdc578
+»
+```
+
+### Logs
+
+To get a detailed log for Mysterium node service, you can use the following command:
+
+```
+sudo journalctl -u mysterium-node.service
+```
+
+## Built-in UI and Node LAN discovery
+
+A Mysterium node starts with a built-in UI server and allows to do the basic configuration.
+
+It uses basic authentication to prevent unauthorized access to the Mysterium node configuration.
+
+Default credentials for built-in UI:
+ - login: `myst`
+ - password: `mystberry`
+
+These credentials can be changed using a built-in UI.
+
+Default port for built-in UI server is `:4449`, and to get access to it you need to open [http://<mysterium-node-ip>:4449](http://<mysterium-node-ip>:4449).
+
+There are several ways to discover a Mysterium Node IP-address.
+
+### Locally hosted Mysterium Node
+
+If you are running a Mysterium node on the same machine that will be used for accessing UI you just need to use the following address:
+
+[http://localhost:4449/](http://localhost:4449/)
+
+### Using a Windows system for accessing Mysterium Node with UI
+
+SSDP based protocol is used for Mysterium Node discovery on Windows systems. Once you have a Mysterium Node with enabled UI running in your local network, it will be possible to see it automatically in the Network section of the Windows Explorer:
+
+<div style="text-align:center">
+  <img src="../../images/postinstall/discovery.jpeg" alt="Login" class="screenshot">
+</div>
+
+You just need to double click on the device icon to open the UI in the default browser.
+
+> Note: Network discovery should be enabled in your Windows.
```

Comment:
I'd point out that in order to use CLI tool on windows user should do it with PowerShell or WSL https://docs.microsoft.com/en-us/windows/wsl/install-win10 since standard CMD does not work